tests/TestCase/I18n/NumberTest.php

```diff
@@ -538,4 +538,22 @@ public function testReadableSizeLocalized()
         $result = $this->Number->toReadableSize(512.05 * 1024 * 1024 * 1024);
         $this->assertEquals('512,05 GB', $result);
     }
+
+    /**
+     * test persisting formatter attributes
+     *
+     * @return void
+     */
+    public function testPersistingFormatterAttributes()
+    {
+        Number::formatter([
+            'type' => 'currency',
+            'locale' => 'en_IN',
+            'pattern' => '¤ #,##,##0',
+            'persistOptions' => true
+        ]);
```

Comment:
@lorenzo Yeah I figured that. I can always make my own formatter instances and reuse them but would be sad that I need to do that in spite of having a utility class which is supposed to help me. Should we add a better way to configure reusable formatter in `Number` class?